src/desktop/components/main_layout/templates/head.jade

```diff
@@ -5,6 +5,24 @@ link( rel='apple-touch-icon', sizes='152x152', href=asset('/images/icon-152.png'
 link( rel='apple-touch-icon', sizes='76x76', href=asset('/images/icon-76.png') )
 link( rel='icon', type='image/png', href=asset('/images/favicon.ico') )
 link( rel='search', type='application/opensearchdescription+xml', href=asset('/images/opensearch.xml'), title='Artsy' )
+
+//- Prefetch major external resources
+link( rel='dns-prefetch', href=asset('https://dpm.demdex.net') )
+link( rel='dns-prefetch', href=asset('https://idsync.rlcdn.com') )
+link( rel='dns-prefetch', href=asset('https://gum.criteo.com') )
+link( rel='dns-prefetch', href=asset('https://api.segment.io') )
+link( rel='dns-prefetch', href=asset('https://d.adroll.com') )
+link( rel='dns-prefetch', href=asset('https://volley.artsy.net') )
+link( rel='dns-prefetch', href=asset('https://api.artsy.net') )
+link( rel='dns-prefetch', href=asset('https://webfonts.artsy.net') )
+link( rel='dns-prefetch', href=asset('https://connect.facebook.net') )
+link( rel='dns-prefetch', href=asset('https://www.googleadservices.com') )
+link( rel='dns-prefetch', href=asset('https://cdn.segment.com') )
+link( rel='dns-prefetch', href=asset('https://googleads.g.doubleclick.net') )
+link( rel='dns-prefetch', href=asset('https://s.adroll.com') )
+link( rel='dns-prefetch', href=asset('https://www.google-analytics.com') )
+link( rel='dns-prefetch', href=asset('https://sb.scorecardresearch.com') )
```

Comment:
You don't need to use the `asset` method for these. Being as they're full urls you should just be able to do `href='https://www.google-analytics.com'`.

Reply:
I think this list is likely too exhaustive and will load some unnecessary resources. Try to focus on the results of new pages... artist, artwork, collect.